buildpipeline/Core-Setup-Ubuntu14.04-x64.json

```diff
@@ -101,7 +101,7 @@
       "value": "PassedViaPipeBuild"
     },
     "CLI_NUGET_FEED_URL": {
-      "value": "https://www.myget.org/F/core-setup-testing"
+      "value": "https://dotnet.myget.org/F/cli-deps"
```

Comment:
That should be removed.  There is no reason why Core-Setup-CrossBuild.json has it.  We are only publishing the `deb-tool` from Ubuntu14.04.